src/components/screen/FindPw.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { ReactElement, useState } from 'react';
+
+import Button from '../shared/Button';
+import { DefaultNavigationProps } from '../../types';
+import { EditText } from '@dooboo-ui/native';
+import { getString } from '../../../STRINGS';
+import styled from 'styled-components/native';
+import { useThemeContext } from '@dooboo-ui/native-theme';
+
+const StyledSafeAreaView = styled.View`
+  flex: 1;
+  background-color: ${({ theme }): string => theme.background};
+`;
+
+const Wrapper = styled.View`
+  margin: 44px;
+`;
+
+const ButtonWrapper = styled.View`
+  width: 100%;
+  margin-top: 20px;
+`;
+
+interface Props {
+  navigation: DefaultNavigationProps<'FindPw'>;
+}
+
+function Page(props: Props): ReactElement {
+  const [email, setEmail] = useState<string>('');
+  const [errorEmail, setErrorEmail] = useState<string>('');
+  const [isFindPw, setIsFindPw] = useState<boolean>(false);
+  let timer: number;
+
+  const { theme } = useThemeContext();
+
+  const validateEmail = (email: string): boolean => {
```

Comment:
Another one here. Should be defined in `utility function` rather than copy and pasting everywhere.